docs/guide/setup.md

```diff
@@ -2,26 +2,60 @@
 
 ## Installation
 
-First, make sure you have postgres installed:
+### macOS
+
+The preferred way to install dbt on macOS is via [Homebrew](http://brew.sh/). Install homebrew, then run:
 
 ```bash
-# linux
-sudo apt-get install libpq-dev python-dev
+brew update
+brew install dbt
 
-# osx
+# to upgrade dbt, use
+brew update
+brew upgrade dbt
+```
+
+If you'd prefer to use the development version of dbt, you can install it as follows. Please note that the development version is considered unstable, and may contain bugs.
+
+```bash
+brew update
+brew install --HEAD dbt
+
```

Comment:
Can you add tiny comments here that indicate that the above is for installing and the below is for updating?